packages/cosmic-swingset/src/export-kernel-db.js

```diff
@@ -41,10 +41,10 @@ export const ExportManifestFileName = 'export-manifest.json';
  */
 export const getEffectiveArtifactMode = artifactMode => {
   switch (artifactMode) {
+    case undefined:
```

Comment:
.. and this might be the flag that one would provide to get a replay-everything -type bundle out of an archival node, to address @gibson042 's comment